docs/designers-developers/developers/filters/README.md

```diff
@@ -4,4 +4,4 @@
 
 There are two types of hooks: [Actions](https://developer.wordpress.org/plugins/hooks/actions/) and [Filters](https://developer.wordpress.org/plugins/hooks/filters/). In addition to PHP actions and filters, WordPress also provides a mechanism for registering and executing hooks in JavaScript. This functionality is also available on npm as the [@wordpress/hooks](https://www.npmjs.com/package/@wordpress/hooks) package, for general purpose use.
 
-You can also learn more about both APIs: [PHP](https://codex.wordpress.org/Plugin_API/) and [JavaScript](/packages/tree/master/packages/hooks).
+You can also learn more about both APIs: [PHP](https://codex.wordpress.org/Plugin_API/) and [JavaScript](https://developer.wordpress.org/block-editor/packages/packages-hooks/).
```

Comment:
This link needs to work also on GitHub:
```suggestion
You can also learn more about both APIs: [PHP](https://codex.wordpress.org/Plugin_API/) and [/packages/hooks/REAMDE.md).
```

Reply:
@nitishkaila thanks for your contribution, can you update the link as suggested above before I merge this PR?

Reply:
It's work on github too. I have already added full link in github. so it's definitely work.

Still if you have any query please let me know.

Reply:
Yes, it does. However please check these document:
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/document.md#using-links

There can be a different version of docs at https://developer.wordpress.org/ because it covers the latest stable version of WordPress and GitHub can be a few weeks ahead.

Reply:
Thank you @gziolo to clarify. I have updated the link.

Reply:
Thanks, I can imagine it's confusing but we operate with a very specific setup where we sync changes to WordPress core through npm packages so it enforces a set of rules to ensure everything works properly :)